### PR TITLE
moved tags related paths into one file so operations prefixes is grouped by files

### PR DESCRIPTION
**Pull request by vfedonkin**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [* ] The title of the PR is clear and informative.
- [* ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [* ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [* ] If applicable, the PR references the bug/issue that it fixes.
- [* ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [* ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [* ] My spec meets the review criteria:
  - [* ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [* ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [* ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by vfedonkin** [comment]
@solankisamir , could you please review this too?
Also, I deleted  apimapisbytags.json, do we need to remove any reference to it in the documentation menu?
Thank you,
Vitaliy.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please address validation errors [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/339176433#L2764)

**Comment by vfedonkin** [comment]
@dsgouda, there are no errors but warnings only, Which one should we fix? PutRequestResponseScheme? It's not possible, since put method receives existing apiId and tagId to make an association between them, but GET method returns all other properties of the API and TAG. Anyway, the result of GET operation can be used as input for PUT, those properties will be just ignored.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json [hunk 1]
```diff
@@ -1246,7 +1246,282 @@
...
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tag_AssignToProduct",
```
Comment (dsgouda) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json]: `PUT` operations must usually be named as `Tag_create` or `Tag_Update`

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json [hunk 2]
```diff
@@ -1246,7 +1246,282 @@
...
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tag_DetachFromProduct",
```
Comment (dsgouda) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json]: `DELETE` operations must be named as `Tag_Delete`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, the naming for certain operations is against recommendations.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json (hunk 1 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json]: The PR was signed off by ARM https://github.com/Azure/azure-rest-api-specs/pull/2257. We are just moving operation into different file, as it caused the Documentation CI to break

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json (hunk 2 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json]: The PR was signed off by ARM https://github.com/Azure/azure-rest-api-specs/pull/2257. We are just moving operation into different file, as it caused the Documentation CI to break

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**